px-map-behavior-layer-IMS.es6.js

```diff
@@ -247,9 +384,18 @@
         this._requestCollectionsFromIMS(options, initialBounds);
       }
 
+      // Setup rtree for storing points for selection
+      this._iconTree = new rbush(16);
+
       //Bind to px-maps moveend to re-request the data with new bounds
       //If layer is not going to be rendered at the current zoom level, don't load
-      this.parentNode.elementInst.on({
+      mapInst.on({
+        zoomstart: () => {
+          // TODO update the transform of the canvas whilst zooming
```

Comment:
Has this TODO been addressed?

Reply:
Not yet - I parked this in the interest of getting the basics integrated first